apps/np-atobarai/src/app/api/webhooks/saleor/transaction-initialize-session/use-case.ts

```diff
@@ -54,6 +54,7 @@ export class TransactionInitializeSessionUseCase extends BaseUseCase {
   protected logger = createLogger("TransactionInitializeSessionUseCase");
   private atobaraiApiClientFactory: IAtobaraiApiClientFactory;
   private appTransactionRepo: TransactionRecordRepo;
+  private goodsBuilder = new NoRefundGoodsBuilder();
```

Comment:
Better specify what is it for, not what it is NOT for.

eg TransactionGoodBuilder?